Kitodo-DataManagement/src/main/java/org/kitodo/data/elasticsearch/search/Searcher.java

```diff
@@ -33,6 +29,10 @@
 import org.kitodo.data.elasticsearch.exceptions.CustomResponseException;
 import org.kitodo.data.exceptions.DataException;
 
+import jakarta.json.Json;
+import jakarta.json.JsonObject;
+import jakarta.json.JsonReader;
+
```

Comment:
Sollten wir "jakarta" in den _config/checkstyle.xml_ ergänzen und als eigene Gruppe vor "java" einsortieren? Die aktuelle Regel kennt es nicht und verlangt daher diese import-Statements ganz am Schluss.